app/models/portfolio_transaction.rb

```diff
@@ -6,6 +6,15 @@ class PortfolioTransaction < ApplicationRecord
   # credit/debit is for stock transactions ie buy/sell stocks
   enum :transaction_type, { deposit: 0, withdrawal: 1, credit: 2, debit: 3, fee: 4 }
 
+  REASONS = {
+    math_earnings: "Earnings from Math",
+    reading_earnings: "Earnings from Reading",
+    attendance_earnings: "Earnings from Attendance",
+    grade_earnings: "Earnings from Grades", # TODO: Remove this
+    transaction_fee: "Transaction Fee",
+    awards: "Award"
```

Comment:
Do we need one more that is 
administrative_adjustments or something like that?  for when there's been an error and we need to add or subtract from an account to "make it right"?